### ISSUE TITLE
Mouseover no longer blue.

### ISSUE THREAD

**Issue by CMR248** [nfdi4plants/nfdi4plants.github.io#14]
For a while the triggered element became "DataPLANT"-blue. This is no longer the case...

**Comment by kMutagene** [comment]
Isn't this what was requested via #3 ?

**Comment by CMR248** [comment]
Good point. I was writing about the icons at the end of the landing page in #3. I think the mouseover color is irrelevant for the branding guidelines.
